src/dmd/dtemplate.d

```diff
@@ -794,6 +794,30 @@ extern (C++) final class TemplateDeclaration : ScopeDsymbol
         return result;
     }
 
+    /******************************
+     * Create a scope for the parameters of the TemplateInstance
```

Comment:
Please use a short summary, an empty new line and then the description.

Reply:
Ah I was wondering why the doc tester was failing.

Reply:
... or not. Seems like html generation is broken?

Reply:
No you broke it with this PR:

```
dmd/dtemplate.d(810): Warning: Ddoc: parameter count mismatch
```

(From the bottom of the log)

Reply:
Eh? I might have been a musician in a previous life, but even _I_ can count to three. Also if thats compiling with "`-Werror`" it would be useful to show it as such. I searched for "Error:" and all I found was a bunch of stuff related to html output, none of which told me where in the source I stuffed up.

Reply:
Well because you used `:` instead of `=` for the Params it couldn't find any and thus it was 0 vs. 3.

Reply:
It's working now: http://dtest.dlang.io/artifact/website-826ea94809d870328bd98d9aeac2472b657d0c49-bd107e0b4d1624be86beb6c8553e79cf/web/phobos-prerelease/dmd_dtemplate.html#.TemplateDeclaration.scopeForTemplateParameters (the URL will be outdated with the next git push / build though)

Reply:
> I searched for "Error:" and all I found was a bunch of stuff related to html output, none of which told me where in the source I stuffed up.

For future reference: scroll down to the end of the log and read the error stack trace from there. That's what I did. Only auto-tester is special in that way that it doesn't show you the errors at the end :/

Reply:
All I'm saying it that the error message should a) be an _error_ message, and b) should tell me that I need to use `=` (or even just `no = found, use = for parameters`.

Reply:
> a) be an error message

Well you can use `-w` to turn warnings into errors (which is what the CI uses) and warnings are an historic relict of DMD anyhow :/

> and b) should tell me that I need to use = (or even just no = found, use = for parameters.

Typically it's a mismatch, but yeah even these parameters aren't shown.
The error message could definitely be improved:

https://github.com/dlang/dmd/blob/a6dad1d2e8ae99afcce5235297407567f96c2c26/src/dmd/doc.d#L297-L300

Reply:
> Well you can use -w to turn warnings into errors

Then it should say `Error:` not `Warning:`. 
I see that CI has failed -> must be an error -> ^F "Error:" -> not helpful -> don't know why has failed.